helper.go

```diff
@@ -114,7 +103,7 @@ func (m *Mux) GetRequestRoute(req *http.Request) string {
 					}
 				}
 			}
-			if r.Match(req) {
+			if ok, _ := r.Match(req); ok {
```

Comment:
Previously, calling this method would leak memory, since it was setting values in the `vars` map (and never clearing them out).